src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/configurationtechnologies/JavaOptions.java

```diff
@@ -0,0 +1,41 @@
+package io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies;
+
+import hudson.Extension;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class JavaOptions extends TechnologyOptions{
```

Comment:
Please add documentation and explain the main goal of this class (Single Role Of Responsibility).